pages/settings/logsAndActivity.vue

```diff
@@ -72,6 +72,7 @@ import Vue from 'vue';
 import { mapActions } from 'vuex';
 import colors from 'vuetify/lib/util/colors';
 import { ActivityLogEntry, LogFile, LogLevel } from '@jellyfin/client-axios';
+import { decodeHTML } from 'entities';
```

Comment:
You need to add the packages to the dependencies with `yarn add entities`.

Currently it's not an explicit dependency of ours. While that's fine for core stuff like Vue and Vuex, this kind of dependency should be explicit.

Reply:
done